weaver/processes/wps_package.py

```diff
@@ -708,6 +718,37 @@ def _update_package_metadata(wps_package_metadata, cwl_package_package):
             set(wps_package_metadata.get("keywords", [])) | set(cwl_package_package.get("s:keywords", []))
         )
 
+    # specific use case with a different mapping
+    if "s:version" in cwl_package_package:
+        wps_package_metadata["version"] = str(
+             str(cwl_package_package.get("s:version", "")) or str(wps_package_metadata.get("version"))
+        )
+
+    for metadata_mapping in SUPPORTED_METADATA_MAPPING:
+        if metadata_mapping in cwl_package_package:
+            metadata = wps_package_metadata.get("metadata", [])
+            if (isinstance((cwl_package_package[metadata_mapping]), str)
+                    and urlparse(cwl_package_package[metadata_mapping]) != ""):
```

Comment:
For consistency with the rest of the code, use : 

```python
if (
    isinstance((cwl_package_package[metadata_mapping]), str)
    and urlparse(cwl_package_package[metadata_mapping]) != ""
):
    [...]
```

Reply:
And, I think it should be `urlparse(cwl_package_package[metadata_mapping]).scheme`, since `urlparse` returns an object, therefore always `!= ""`. 

Try for example with `{"s:dateCreated": "2024-08-29"}`. It should enter this `if`, when the `else` is expected.